components/internal/modals/EditDoctorInformationModal.tsx

```diff
@@ -44,7 +44,7 @@ export default function EditDoctorInformationModal() {
       >
         <ModalOverlay />
         <form onSubmit={handleSubmit}>
-          <ModalContent paddingX="40px">
+          <ModalContent paddingX="36px">
             <ModalHeader paddingBottom="12px" paddingTop="24px" paddingX="4px">
```

Comment:
`paddingX` of the `ModalContent` was changed to "36px" because the total `paddingX` of the modal text should be 40px which is the sum of the `paddingX` of the `ModalContent` and  `paddingX` of the `ModalHeader`/`ModalBody`/`ModalFooter`